src/python/pants/bsp/protocol.py

```diff
@@ -4,8 +4,7 @@
 
 import logging
 from concurrent.futures import Future
-from dataclasses import dataclass
-from typing import Any, BinaryIO, Callable, Type
+from typing import Any, BinaryIO, ClassVar, Type
```

Comment:
fyi you can use lowercase `type`. Only time you can't is if it's used in actual code, like `class Foo(Type)` or `MyAlias = Type`.

Reply:
Done.